docs/PID tuning.md

```diff
@@ -29,6 +29,24 @@ changing slowly (so the P and I terms aren't having enough impact on reaching th
 in the correction in order to reach the target sooner. If the error is rapidly converging to zero, the D term causes the
 strength of the correction to be backed off in order to avoid overshooting the target.
 
+##TPA and TPA Breakpoint
```

Comment:
You may be correct.  My goal is to introduce some basic documentation in Cleanflight for TPA and TPA breakpoint.  I would welcome additional work on this document to be more accurate.  You're elected!